qadence/mitigations/analog_zne.py

```diff
@@ -11,13 +11,29 @@
 from qadence.backends.api import backend_factory
 from qadence.backends.pulser.backend import Backend
 from qadence.blocks import block_to_tensor
+from qadence.blocks.utils import expression_to_uuids
 from qadence.measurements import Measurements
 from qadence.mitigations import Mitigations
 from qadence.noise import Noise
+from qadence.parameters import Parameter
 from qadence.utils import Endianness
 
 
-def zne(noise: Noise, zne_datasets: list) -> list:
+def zne_pulse(stretches: dict, zne_datasets: list, zne_value: Tensor) -> list:
```

Comment:
Could you make a more explicit typing? `list[?]` and `dict[?, ?]`.

Reply:
If stretches can be None, this should be in the function args definition.

Reply:
Changed to tensors.